pkg/frontend/querymiddleware/limits_test.go

```diff
@@ -295,36 +342,50 @@ func TestLimitsMiddleware_MaxQueryLength(t *testing.T) {
 
 	for testName, testData := range tests {
 		t.Run(testName, func(t *testing.T) {
-			req := &PrometheusRangeQueryRequest{
-				start: util.TimeToMillis(testData.reqStartTime),
-				end:   util.TimeToMillis(testData.reqEndTime),
+			reqs := map[string]MetricsQueryRequest{
+				"range query": &PrometheusRangeQueryRequest{
+					start: util.TimeToMillis(testData.reqStartTime),
+					end:   util.TimeToMillis(testData.reqEndTime),
+				},
+				"remote read": &remoteReadQueryRequest{
+					path:      remoteReadPathSuffix,
+					promQuery: "",
```

Comment:
nit: redundant

Reply:
Removed in https://github.com/grafana/mimir/pull/8374/commits/14101afef8c91990d21e06b402d3cea4a75671bf